site/src/components/AuditLogRow/AuditLogDescription/AuditLogDescription.test.tsx

```diff
@@ -7,9 +7,13 @@ import {
   MockAuditLogUnsuccessfulLoginUnknownUser,
 } from "testHelpers/entities"
 import { AuditLogDescription } from "./AuditLogDescription"
-import { AuditLogRow } from "./AuditLogRow"
-import { render } from "../../testHelpers/renderHelpers"
+import { AuditLogRow } from "../AuditLogRow"
```

Comment:
These tests aren't perfect - I was having difficulty testing strings with an embedded `Link`. I will return at a later time and maybe turn these into stories or create a custom matcher.